clients/go-mobile/client.go

```diff
@@ -0,0 +1,72 @@
+// SPDX-FileCopyrightText: 2022 ANSSI
+// SPDX-License-Identifier: Apache-2.0
+
+package gomobile
+
+import (
+	"crypto/rsa"
+	"math/big"
+
+	"github.com/fxamacker/cbor/v2"
+	"github.com/google/go-attestation/attest"
+)
+
+type CredentialBlob struct {
+	Secret     []byte
+	Cred       []byte
+	CredSecret []byte
+}
+
+func createRSAPublicKey(publicBytes []byte, exponent int) rsa.PublicKey {
+	public := big.NewInt(0).SetBytes(publicBytes)
+	key := rsa.PublicKey{
+		N: public,
+		E: exponent,
+	}
+	return key
+}
+
+func MakeCredential(ekn []byte, eke int, encodedAP []byte) *CredentialBlob {
```

Comment:
I'm confused why the AK (attestation KEY), which is called as such in the Android Code, is called AP (attestation parameters) here. Even more confusing is the `AK: ap` line below. I'm not saying you're doing anything wrong, it seems to match the naming convention of go-tpm, but it's confusing: can you please add to the function comment a note explaining what AP means and why AK==AP (or, if there's a subtle difference, what the difference actually is)?

Reply:
There is effectively a subtle difference:
- Ak is an actual attestation key
- Activation parameters contains several information, such as the public key itself, but also what we call it's name, which (if I remember well) is a hash of it's public part and it's template.

I'll indeed add a comment or change variable names to make that less confusing